app/services/landingPageTourService.ts

```diff
@@ -0,0 +1,120 @@
+import * as angular from 'angular';
+import * as jQuery from 'jquery';
+
+interface ILandingPageTourService {
+  startTour(tourConfig: any);
+}
+
+export class LandingPageTourService implements ILandingPageTourService {
+  public static $inject = ['$document', '$timeout', 'Constants', 'GuidedTourService'];
+
+  private $document: any;
+  private $timeout: any;
+  private Constants: any;
+  private GuidedTourService: any;
+  private tourConfig: any;
+
+  constructor ($document: any, $timeout: any, Constants: any, GuidedTourService: any) {
+    this.$document = $document;
+    this.$timeout = $timeout;
+    this.Constants = Constants;
+    this.GuidedTourService = GuidedTourService;
+
+    this.tourConfig = {
+      id: "getting-started-tour",
+      steps: []
+    };
+
+    let stepConstants = Constants.LANDING_PAGE_TOUR_STEPS || {};
```

Comment:
There are two pieces that make it difficult for steps to be added. The target field which specifies a jquery selector to find the target component, and any actions that need to be performed prior to showing the step (ie. open the Languages tab in the catalog).